Deserialize.scala

```diff
@@ -0,0 +1,167 @@
+// Author: James Dunn, UC Berkeley (dunn@eecs.berkeley.edu)
+
+package FFTGenerator
+
+import chisel3._
+import chisel3.experimental.FixedPoint
+import chisel3.util.Decoupled
+import chisel3.util._
+import dspjunctions._
+import dsptools.numbers._
+import freechips.rocketchip.diplomacy.LazyModule
+import freechips.rocketchip.subsystem.BaseSubsystem
+import scala.math.sqrt
+
+/**
+  * This block performs a deserialization. It takes time-series inputs on a
+  * decoupled interface. These inputs are collected into a vector of registers.
+  * This vector of registers is mapped to an output vector IO. When a counter
+  * reaches the deserialization factor, output.valid is asserted.
+  */
+
+/**
+	* Base class for Deserialize parameters.
+	* These are type-generic.
+	*/
```

Comment:
Looks like the files have some funky spacing issues. Can you clean this up.

Reply:
looks like there were tabs instead of spaces in some places. VSCode rendered the tabs identically to spaces but git (and therefore github) didn't.